openapi-v1.yaml

```diff
@@ -7041,8 +7093,10 @@ paths:
       description: Accepts invitation
       operationId: acceptInvitation
       responses:
-        204:
+        200:
```

Comment:
This breaks compatibility. Probably nobody will notice but consider adding a new route